hw/ip/otbn/dv/otbnsim/stepped.py

```diff
@@ -252,13 +257,24 @@ def on_print_call_stack(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
     return None
 
 
-def on_edn_rnd_data(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
+def on_edn_step(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
     if len(args) != 1:
-        raise ValueError('edn_rnd_data expects exactly 1 argument. Got {}.'
+        raise ValueError('edn_step expects exactly 1 argument. Got {}.'
+                         .format(args))
+
+    edn_rnd_data = read_word('edn_step', args[0], 32)
+
+    sim.state.step_edn(edn_rnd_data)
+
+    return None
+
+
+def on_edn_rnd_cdc_done(sim: OTBNSim, args: List[str]) -> Optional[OTBNSim]:
+    if len(args) != 0:
+        raise ValueError('edn_rnd_cdc_done expects exactly 0 argument. Got {}.'
```

Comment:
Nit: "expects no arguments" maybe? (Or otherwise "exactly 0 arguments", but that's probably less idiomatic).